e2e/cypress/integration/system_console/reporting/site_statistics_spec.js

```diff
@@ -12,56 +12,63 @@
 import * as TIMEOUTS from '../../../fixtures/timeouts';
 
 describe('System Console > Site Statistics', () => {
-    it('MM-T903 - Site Statistics > Deactivating a user increments the Daily and Monthly Active Users counts down', () => {
-        cy.apiInitSetup().then(({team, user}) => {
-            const testUser = user;
+    let testUser;
+    let totalActiveUsersInitial;
+    let dailyActiveUsersInitial;
+    let monthlyActiveUsersInitial;
+    let totalActiveUsersFinal;
+    let dailyActiveUsersFinal;
+    let monthlyActiveUsersFinal;
 
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
             // # Login as test user and visit town-square
+            testUser = user;
             cy.apiLogin(testUser);
             cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+    it('MM-T903 - Site Statistics > Deactivating a user increments the Daily and Monthly Active Users counts down', () => {
+        // # Post a message as testUser to make them daily/monthly active
+        cy.getCurrentChannelId().then((channelId) => {
+            const message = 'New Daily Message';
+            cy.postMessageAs({sender: testUser, message, channelId});
+        });
 
-            // # Go to admin console
-            goToAdminConsole();
-
-            // # Go to system analytics
-            cy.findByTestId('reporting.system_analytics', {timeout: TIMEOUTS.ONE_MIN}).click();
-            cy.wait(TIMEOUTS.ONE_SEC);
+        // # Go to admin console
+        goToAdminConsole();
 
-            let totalActiveUsersInitial;
-            let dailyActiveUsersInitial;
-            let monthlyActiveUsersInitial;
-            let totalActiveUsersFinal;
-            let dailyActiveUsersFinal;
-            let monthlyActiveUsersFinal;
+        // # Go to system analytics
+        cy.findByTestId('reporting.system_analytics', {timeout: TIMEOUTS.ONE_MIN}).click();
+        cy.wait(TIMEOUTS.ONE_SEC);
 
-            // # Get the number and turn them into numbers
-            cy.findByTestId('totalActiveUsers').invoke('text').then((text) => {
-                totalActiveUsersInitial = parseInt(text, 10);
-                cy.findByTestId('dailyActiveUsers').invoke('text').then((text2) => {
-                    dailyActiveUsersInitial = parseInt(text2, 10);
-                    cy.findByTestId('monthlyActiveUsers').invoke('text').then((text3) => {
-                        monthlyActiveUsersInitial = parseInt(text3, 10);
+        // # Get the number text and turn them into numbers
```

Comment:
Hi @prapti I think we can cleanup this nested lines. I tried locally and the following works for me,
```
        // # Get the number text and turn them into numbers
        cy.findByTestId('totalActiveUsers').invoke('text').then((totalActiveText) => {
            totalActiveUsersInitial = parseInt(totalActiveText, 10);
        });
        cy.findByTestId('dailyActiveUsers').invoke('text').then((dailyActiveText) => {
            dailyActiveUsersInitial = parseInt(dailyActiveText, 10);
        });
        cy.findByTestId('monthlyActiveUsers').invoke('text').then((monthlyActiveText) => {
            monthlyActiveUsersInitial = parseInt(monthlyActiveText, 10);

            // # Deactivate user and reload page and then wait 2 seconds
            cy.externalActivateUser(testUser.id, false);
            cy.reload();
            cy.wait(TIMEOUTS.TWO_SEC);
        });
        // # Get the numbers required again
        cy.findByTestId('totalActiveUsers').invoke('text').then((totalActiveFinalText) => {
            totalActiveUsersFinal = parseInt(totalActiveFinalText, 10);
        });
        cy.findByTestId('dailyActiveUsers').invoke('text').then((dailyActiveFinalText) => {
            dailyActiveUsersFinal = parseInt(dailyActiveFinalText, 10);
        });
        cy.findByTestId('monthlyActiveUsers').invoke('text').then((monthlyActiveFinalText) => {
            monthlyActiveUsersFinal = parseInt(monthlyActiveFinalText, 10);

            // * Assert that the final number is the initial number minus one
            expect(totalActiveUsersFinal).equal(totalActiveUsersInitial - 1);
            expect(dailyActiveUsersFinal).equal(dailyActiveUsersInitial - 1);
            expect(monthlyActiveUsersFinal).equal(monthlyActiveUsersInitial - 1);
        });
```